package.json

```diff
@@ -8,8 +8,10 @@
     "node": "4.1.x"
   },
   "scripts": {
-    "start": "node index.js",
-    "test": "mocha test/*_test.js test/**/*_test.js"
+    "start": "node -r babel-core/register -r dotenv/config lib/index.js",
```

Comment:
What was your reasoning for moving these here rather than keeping them at the top of an index.js file that then module.exports lib/index.js? We'd then be able to include the polyfill in there as well.